Changed HDFSConfigException

milesgranger opened the pull request:
Just allowing user to override the autodetected IP/Ports without raising HDFSConfigException

milesgranger committed:
Changed HDFSConfigException

Allows user to override the autodetected IP/Ports without raising HDFSConfigException

quasiben commented:
@milesgranger Thanks for the PR!  Perhaps we should consider a flag in the constructor: `ignore_exceptions` ?

milesgranger commented:
@quasiben Probably a better solution indeed, just thought I'd share what I did to make it work for me. I'll get back to you with the updated PR. :)

milesgranger committed:
added validate flag to constructor

validation flag to bypass validation of IP/Ports, this avoids raising an error when these do not match config file.

quasiben commented:
Looks good.  Thanks for the PR!
